testsuite/gnat2goto/tests/floating_point_constant/floating_point_constant.adb

```diff
@@ -0,0 +1,17 @@
+--  This test is testing the floating point constants
+procedure Floating_Point_Constant is
+  F : Float := 10.0;
+  --  the function here is needed because for
+  --  the second assert the constant gets simplified
+  --  into True that doesn't have a value at the moment.
+  function Haha return Integer is
+   (1);
+begin
+  if F < 5.0 then
+    pragma Assert (False);
+  end if;
+
```

Comment:
Noticed that in previous test there is  --  CBMC: SUCCESS / FALSE prior to each check.